core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -50,16 +42,56 @@ describe('Upgrade test', function () {
     let forceDeployBytecode: string;
     let logs: fs.WriteStream;
 
+    let ethProviderAddress: string | undefined;
+    let contractsL2DefaultUpgradeAddr: string;
+    let deployerAddress: string;
+    let complexUpgraderAddress: string;
+
     before('Create test wallet', async () => {
-        tester = await Tester.init(process.env.CHAIN_ETH_NETWORK || 'localhost');
+        forceDeployAddress = '0xf04ce00000000000000000000000000000000000';
+        deployerAddress = '0x0000000000000000000000000000000000008007';
+        complexUpgraderAddress = '0x000000000000000000000000000000000000800f';
+
+        if (fileConfig.loadFromFile) {
+            const contractsConfig = loadConfig({ pathToHome, chain: fileConfig.chain, config: 'contracts.yaml' });
+            const secretsConfig = loadConfig({ pathToHome, chain: fileConfig.chain, config: 'secrets.yaml' });
+
+            ethProviderAddress = secretsConfig.l1.l1_rpc_url;
+            contractsL2DefaultUpgradeAddr = contractsConfig.l2.default_l2_upgrader;
+            process.env.CONTRACTS_DEFAULT_UPGRADE_ADDR = contractsConfig.l1.default_upgrade_addr;
+            process.env.CONTRACTS_PRIORITY_TX_MAX_GAS_LIMIT = '72000000';
```

Comment:
Don't use envs inside tests.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2515/commits/f7c285e6db84b64769a2d4453c1c00789152fa6c